### DISCUSSION TITLE
coursera: Deep Learning - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#115] [Announcements]
# coursera: Deep Learning - CS%E8%87%AA%E5%AD%A6%E6%8C%87%E5%8D%97

CS自学指南

https://csdiy.wiki/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/CS230/

**Comment by habaohaba** [comment]
想问cousera上的这个specialization建议购买吗？我之前学机器学习的课，可以直接学习，只是证书需要购买。这个specialization做作业之类的也需要升级为付费用户。想问大家觉得值得购买吗?

**Reply by ballin0105** [reply]
有关省钱的东西都可以考虑淘b

**Reply by MTDickens** [reply]
建议申请助学金

**Reply by habaohaba** [reply]
更新一下，我买了一个月，然后一个月上完了deep learning specialization，感觉还算值得吧，但是吴恩达的这几个课，课件都做的不是很好，全是手写的，不太好做笔记或者回过头复习

**Comment by W01fh4cker** [comment]
补充一下，B站搬运的视频的地址：`https://www.bilibili.com/video/BV1A5411b7w7/`

**Comment by RunningVic** [comment]
请问在学习深度学习之前，需要先学机器学习吗？

**Reply by junle-chen** [reply]
感觉不需要，但是需要一些数学基础

**Comment by asdfefbhjy** [comment]
https://d2l.ai/

**Comment by linyifan0223** [comment]
这个课程对小白友好吗

**Reply by waterEand** [reply]
很友好

**Comment by zhou427** [comment]
这个和李宏毅的怎么选？
